shared/media-accelerator/develop/_ensure-channel-quality.mdx

```diff
@@ -0,0 +1,101 @@
+import * as data from '@site/data/variables';
+import Introduction from '@docs/shared/video-sdk/common/channel_quality_intro.mdx';
+import Prerequisites from '@docs/shared/common/prerequities.mdx';
+import ProjectSetup from '@docs/shared/media-accelerator/develop/ensure-channel-quality/project-setup/index.mdx';
+import ProjectImplement from '@docs/shared/media-accelerator/develop/ensure-channel-quality/project-implementation/index.mdx';
+import ProjectTest from '@docs/shared/media-accelerator/develop/ensure-channel-quality/project-test/index.mdx';
+import Reference from '@docs/shared/media-accelerator/develop/ensure-channel-quality/reference/index.mdx';
+
+<Introduction />
+
+This page shows you how to use <Vpd k="SDK"/> features to account for these factors and ensure optimal audio and video quality in your {props.PRODUCT} <Vpl k="CLIENT" />.
+
+## Understand the tech
+
+<Vpd k="SDK"/> provides the following features to deal with channel quality issues:
+
+- **Choose an audio codec to optimize bit rate and quality**
+
+  In audio and video streaming, the choice of an encoding algorithm affects both quality and bit rate. Your goal is to lower the required bit rate while maintaining quality at the desired level. <Vpd k="SDK" /> offers the following built-in audio codecs to choose from:
+
+  - Opus
+  - G722
+  - G711A (PCMU) 
+  - G711U (PCMU) 
+
+  You specify an audio codec when you join a channel. You also set the sampling rate and the number of channels at this time, depending on your audio quality requirements. If your <Vpl k="CLIENT"/> needs a custom encoding algorithm, you disable use of a built-in codec and implement your own encoder. 
+
+- **Adjust sending bit rate in real time**
+
+  In order to optimize data transmission and avoid network congestion, <Vpd k="SDK"/> suggests adjusting the sending bit rate in real-time according to changes in network conditions.
```

Comment:
The SDK suggests? 

Best practice is to ...

Reply:
fixed